tests/test_meshtastic_utils.py

```
@@ -4704,14 +4693,13 @@ def test_disconnect_ble_by_address_unknown_is_connected_defaults_false(
-        async def _noop(*_args, **_kwargs):
+        def _noop(*_args: object, **_kwargs: object) -> None:
             """
-            Asynchronous no-op that ignores all positional and keyword arguments.
+            No-operation callable that accepts and ignores all positional and keyword arguments.
 
-            Returns:
-                None: Always returns None.
+            Used as a generic placeholder callback or side-effect (for example, in mocks, timers, or disconnect helpers).
             """
-            return None
+            pass
```

Ruff (0.15.10)

[warning] Unnecessary `pass` statement

Remove unnecessary `pass`

(PIE790)